src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -77,50 +78,119 @@ public void Cleanup()
             CleanupTestOutput();
         }
 
-        [TestMethod]
+        [DataTestMethod]
+        [DataRow(true)]
+        [DataRow(false)]
         [Timeout(30000)]
-        public void Scan_Integration_WildlifeManager()
+        public void Scan_Integration_WildlifeManager(bool sync)
         {
-            WindowScanOutput results = Scan_Integration_Core(WildlifeManagerAppPath, WildlifeManagerKnownErrorCount);
+            WindowScanOutput results = Scan_Integration_Core(sync, WildlifeManagerAppPath, WildlifeManagerKnownErrorCount);
             EnsureGeneratedFileIsReadableByOldVersionsOfAxeWindows(results, TestProcess.Id);
         }
 
-        // [TestMethod]
+        // [DataTestMethod]
+        //[DataRow(true)]
+        //[DataRow(false)]
```

Comment:
super-nit, but can we insert a space after the `//` in these lines?
```suggestion
        // [DataRow(true)]
        // [DataRow(false)]
```